### PR TITLE
Model for Academy Award Nominees

### PR DESCRIPTION
**Pull request by annebyrne**

### CODE REVIEW

**Comment by shtukas** [comment]
:+1:

**Comment by shtukas** [comment]
:+1:

**Comment by tomrf1** [comment]
👍

File: src/main/scala/com/gu/facebook_news_bot/models/UserNoms.scala [hunk 1]
```diff
@@ -5,3 +5,49 @@ case class UserNoms(ID: String,
                     bestDirector: Option[String] = None,
                     bestActor: Option[String] = None,
                     bestActress: Option[String] = None)
+
+case class IndividualNominee(name: String, pictureUrl: String)
```
Comment (LATaylor-guardian) [inline src/main/scala/com/gu/facebook_news_bot/models/UserNoms.scala]: `case class IndividualNominee(name: String, pictureUrl: String = "https://media.guim.co.uk/82ef416b00febc774665554973c2965c7f82b819/0_191_2266_1360/140.jpgq")`

Then you don't need to write defaultPictureUrl everywhere where you are instantiating `IndividualNominee` :)

**Pull request merged**